lib/app/modules/settings/views/settings_view.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:managment_app/Themes/themes.dart';
+import 'package:managment_app/components/SettingButton.dart';
+import '../../../../components/Logoutbutton.dart';
+import '../controllers/settings_controller.dart';
+
+class SettingsView extends GetView<SettingsController> {
+  const SettingsView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    return Scaffold(
+      body: Stack(
+        children: [
+          Container(
+            decoration: BoxDecoration(
+              gradient: LinearGradient(
+                colors: [theme.colorLevel1, theme.colorLevel2],
+                begin: Alignment.topCenter,
+                end: Alignment.bottomCenter,
+              ),
+            ),
+          ),
+          SingleChildScrollView(
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
+                Stack(
+                  alignment: Alignment.center,
+                  children: [
+                    Container(
+                      height: 150,
+                      decoration: BoxDecoration(
+                        color: theme.colorLevel2,
+                      ),
+                    ),
+                    Positioned(
+                      top: -10,
+                      child: Container(
+                        padding: const EdgeInsets.symmetric(
+                            vertical: 20, horizontal: 60),
+                        decoration: BoxDecoration(
+                          color: theme.colorLevel1,
+                          borderRadius: BorderRadius.circular(20),
+                          boxShadow: [
+                            BoxShadow(
+                              color: Colors.black.withOpacity(0.1),
+                              spreadRadius: 5,
+                              blurRadius: 10,
+                              offset: Offset(0, 5),
+                            ),
+                          ],
+                        ),
+                        child: Text(
+                          "Settings",
+                          style: GoogleFonts.raleway(
+                            fontSize: 20,
+                            letterSpacing: 4,
```

Comment:
use theme that are already defined in themes, they have same fontStyle, you can use .copywith to change the individual parameters.

make this change everywhere